doc/source/user_guide/visualization.rst

```diff
@@ -1641,3 +1641,45 @@ when plotting a large number of points.
    :suppress:
 
     plt.close('all')
+
+Plotting backends
+-----------------
+
+Starting in 0.25 pandas can be extended with third-party plotting backends. The
+main idea is letting users select a plotting backend different than the provided
+one based on Matplotlib. For example:
+
+.. code-block:: python
+
+    >>> pd.set_option('plotting.backend', 'backend.module')
+    >>> pd.Series([1, 2, 3]).plot()
```

Comment:
Another way to use it is with the `backend=` keyword in the `plot()` call, so I would also show that.

Reply:
I tried to use that locally but couldn't actually figure out how to do that.
Can you please elaborate a bit more on how to use it that way?

Reply:
The PR that introduced this is https://github.com/pandas-dev/pandas/pull/28622. I never used it myself, but from that PR it seems you can `backend="name"`. I would recommend to install one of the existing backends to test.

Reply:
Can you tackle this comment?

Reply:
I couldn't understand what do you mean by "Can you tackle this comment?". Can you please elaborate on this?

Reply:
See my suggestion above: I think we need to include an explanation that you can also do `plot(.., backend="..")` instead of setting the option

Reply:
I have already added that, see the lines 1657 to 1663.

Reply:
On those lines you explain an alternative way to set the option, but I mean a keyword argument to the function: `df.plot(backend="...")`. See the PR I linked to above.